docs/admin/receipt_template_editor.adoc

```diff
@@ -228,15 +232,14 @@ once you import the receipts on the new machine.
 
 Exporting templates
 ^^^^^^^^^^^^^^^^^^^
-As you can only save a template on to the computer you are working on you will
+As you can only save a template on to the computer you are working (*Save Locally*) on you will
 need to export the template if you have more than one computer that prints out
 receipts (i.e., more than one computer on the circulation desk, or another
 computer in the workroom that you use to checkin items or capture holds with)
 
-. Export.  
-. Select the location to save the template to, name the template, and click
-*Save*.
-. Click OK. 
+. Export Customized Templates.
+. Saves a “print_templates.json” file in the downloads folder.
+NOTE: You can rename a the file to better track your changes.
```

Comment:
Need to remove extra word "a" from "You can rename a the file".

Reply:
Corrected ths.